Assets/Scripts/Actors/ActorComponents/PlayerActorPhysics.cs

```diff
@@ -1,135 +0,0 @@
-﻿using UnityEngine;
-using System.Collections;
-
-[RequireComponent( typeof( PlayerActor ) )]
-public class PlayerActorPhysics : ActorPhysics
-{
-	PlayerActor _actor;
-	bool _jumpButtonDown = false;
-	bool _isGrabbing = false;
-
-	public override void Awake()
-	{
-		base.Awake();
-
-		_actor = GetComponent<PlayerActor>();
-		ChangeState( ActorStates.Grounded );
-	}
-
-	void Update()
-	{
-		if ( Input.GetKeyDown( KeyCode.Escape ) )
```

Comment:
Yeah, once we have some kind of level manager or something I'd say that's the place to put it. Or if we have something handling pausing and whatnot.